docker/postgres/init.sql

```diff
@@ -11,3 +11,6 @@ CREATE USER taskcluster_queue PASSWORD 'queue_password';
 CREATE USER taskcluster_secrets PASSWORD 'secrets_password';
 CREATE USER taskcluster_web_server PASSWORD 'web_server_password';
 CREATE USER taskcluster_worker_manager PASSWORD 'worker_manager_password';
+
+CREATE DATABASE "taskcluster-test";
+GRANT ALL ON DATABASE "taskcluster-test" TO taskcluster;
```

Comment:
This is an accidental addition to this PR - create separate database in local environment for tests and generation.

Problem is if you develop with single database locally and then run `yarn generate` - it would wipe out whole database. 

`export TEST_DB_URL=postgresql://postgres@localhost:5432/taskcluster-test` for `yarn generate` avoids this

Reply:
Does this get run on production deployments too?

Reply:
Do we need to update any of the developer docs to explain how multiple local databases should be used, and any local config that should be applied for a working system?